docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -14,6 +14,13 @@ See [Create cluster workflow]({{< relref "../../concepts/clusterworkflow" >}}) f
 To create an EKS Anywhere cluster you will need [`eksctl`](https://eksctl.io) and the `eksctl-anywhere` plugin.
 This will let you create a cluster in multiple providers for local development or production workloads.
 
+To create an EKS Anywhere cluster you will need [`eksctl`](https://eksctl.io), the `eksctl-anywhere` plugin, `kubectl`, `Docker` and `Helm`. This will let you create a cluster in multiple providers for local development or production workloads, and deploying workloads to your clusters. 
```

Comment:
I'm not sure we should add all these in this getting started doc kubectl + docker (though i think that is already called out somewhere?) since most arent needed for the typical flow. I dont have a problem for calling these out where they are used, like in the troubleshooting sections or what not.

I am not sure where helm/flux/kind are needed?  Those are bundled with eks-a via the cli-tools image which is automatically pulled down during the execution of the create cluster flow.

Reply:
@gobrial is working on adding EKS-A workshop content to the EKS-A docs. He's looking for opportunities add content to the existing docs to support the workshop so whole procedures don't have to be repeated. 

In this case, I agree with @jaxesn . I'm inclined to add directions to installing tools in the sections where they are used, since they are not needed for initial cluster creation.

Reply:
I removed the binaries contents, but added a new page for the preparation of VSphere